x-pack/plugins/security_solution/public/management/pages/event_filters/store/middleware.ts

```diff
@@ -136,20 +136,6 @@ const eventFiltersUpdate = async (
       getNewComment(store.getState())
     ) as UpdateExceptionListItemSchema;
 
-    // Clean unnecessary fields for update action
-    [
-      'created_at',
-      'created_by',
-      'created_at',
-      'created_by',
-      'list_id',
-      'tie_breaker_id',
-      'updated_at',
-      'updated_by',
-    ].forEach((field) => {
-      delete updatedCommentsEntry[field as keyof UpdateExceptionListItemSchema];
```

Comment:
Moved from the middleware to the service as the service is used in other places (with react-query)